packages/ui/utils/formatNumber.ts

```diff
@@ -20,3 +20,7 @@ export const bigMul = (value1: string, value2: string) => {
   const val = multiply(bignumber(value1), bignumber(value2));
   return format(val, { notation: 'fixed' });
 };
+
+export const toCeil = (value: number, len: number) => {
```

Comment:
Nice. 👍 
Sad thought that we don't have a test suite to add some unit test for this function.

Reply:
I will add it later on